libs/cuvslam/cuvslam2.cpp

```
@@ -595,15 +810,26 @@ std::optional<Odometry::Gravity> Odometry::GetLastGravity() const {
 std::optional<Odometry::ImuState> Odometry::GetImuState() const {
   THROW_INVALID_ARG_IF(!impl->imu_fusion_enabled, "IMU fusion is disabled");
 
-  const auto s = static_cast<odom::StereoInertialOdometry*>(impl->visual_odometry.get())->GetImuState();
+  if (impl->odometry_mode == OdometryMode::Inertial) {
+    const auto s = static_cast<odom::StereoInertialOdometry*>(impl->visual_odometry.get())->GetImuState();
```

Clang (14.0.6)

[warning] variable name 's' is too short, expected at least 3 characters

(readability-identifier-length)